projects/cps-ui-kit/src/lib/components/cps-scheduler/cps-scheduler.component.ts

```diff
@@ -180,7 +180,9 @@ export class CpsSchedulerComponent implements OnInit, OnChanges {
   selectOptions = this._getSelectOptions();
   timeZoneOptions = timeZones.map((tz) => ({ label: tz, value: tz }));
   state: any;
-  form!: UntypedFormGroup;
+  form: UntypedFormGroup = this._fb.group({
+    advanced: ['', [this._validateAdvancedExpr]]
+  });
```

Comment:
```suggestion
  form: FormGroup<{ advanced: FormControl<string> }> = this._fb.group({
    advanced: ['', [this._validateAdvancedExpr]]
  });
```
good opportunity to type this form (we might wanna get rid of all untyped forms in the future for better type safety)

Reply:
@lukasmatta  I tried you suggestion but it does not workout so I just gave it the FormGroup type: ✖ Compiling with Angular sources in Ivy partial compilation mode.
ERROR: projects/cps-ui-kit/src/lib/components/cps-scheduler/cps-scheduler.component.ts:183:3 - error TS2322: Type 'FormGroup<{ advanced: FormControl<(string | ((c: FormControl<any>) => { invalidExpression: string; } | null)[])[] | null>; }>' is not assignable to type 'FormGroup<{ advanced: FormControl<string>; }>'.
  Types of property 'controls' are incompatible.
    Type '{ advanced: FormControl<(string | ((c: FormControl<any>) => { invalidExpression: string; } | null)[])[] | null>; }' is not assignable to type '{ advanced: FormControl<string>; }'.

Reply:
using typed FormBuilder as you did is correct, good job!